tests/serialization/test_serialization.py

```diff
@@ -26,6 +26,18 @@ class User(AvroModel):
     addresses: typing.List[Address]
 
 
+@dataclass
+class UserCompatible(AvroModel):
+    "User with multiple Address"
+    name: str
+    age: int
+    addresses: typing.List[Address]
+    nickname: typing.Optional[str] = None
+
+    class Meta:
+        schema_name = "User"
```

Comment:
I think it's valuable to allow for a custom schema name, different than the default class' name. Especially, when one has two schemas defined for the same resource - it's not possible two have two classes with the same name.

It might occur, for example, if a system has some schema registry, that contains different versions of one schema, and those schemas have to be represented in the code - in this case, using **dataclasses-avroschema**.

Reply:
It's not a must-have though. I can drop this feature if this is not the right direction for **dataclasses-avroschema**.

Reply:
No, it is fine. I thought it was related to the `deserialization` process but actually is a new feature. Let's keep it